README.rst

```diff
@@ -2,5 +2,7 @@
 github-experiments
 ==================
 
-How does github behave?
+How does github behave, and why do I have to worry about mutation and
+inconsistently changing UI when the whole point of DRCS is to have
+locally verifiable append-only distributable data structures?
```

Comment:
This is probably because of business-capture, network effects, and the competitive simplicity of centralized solutions.